course-v2/layouts/partials/course_description.html

```diff
@@ -3,21 +3,25 @@
 {{ with $courseData.course_description }}
   {{ $shouldCollapseDescription = gt (len .) 320 }}
 {{ end }}
-
-<div class="course-description">
+{{ $truncatedDescription := $courseData.course_description }}
+{{ $expandedDescription := print $courseData.course_description " [Show less](#collapse_description)" }}
+{{ if $shouldCollapseDescription }}
+  {{ $truncatedDescription = truncate 320 "...[Show more](#expand_description)" $courseData.course_description }}
```

Comment:
**Requested Change:** IMO "Show more" / "Show less". should but a button styled as a link, not a link. I realize this is a little bit of a pain to style, but we might want such link-styled buttons else. Maybe a link_button shortcode or partial would be useful? Also, using a link has consequences for the URL:


https://user-images.githubusercontent.com/9010790/193831417-8ea04d0f-dcce-4361-ad25-a37379ac8a5e.mov



_If we do need to use a link, there are probably ways to work around the URL issue._